src/components/Participant/ParticipantCard.js

```diff
@@ -3,20 +3,19 @@ import styled from 'styled-components';
 export default function ParticipantCard({name, businessType, address, liveDate, precinct, handleFlyTo}) {
   return (
     <ParticipantButton onClick={handleFlyTo}>
-      <dl>
-        <h2>{name}</h2>
-        <dt>Business Type: {businessType}</dt>
-        <dt>Address: {address}</dt>
-        <dt>Since: {liveDate}</dt>
-        <dt>Precint: {precinct}</dt>
-      </dl>
+      <h2>{name}</h2>
+      <p>Business Type: {businessType}</p>
+      <p>Address: {address}</p>
+      <p>Since: {liveDate}</p>
+      <p>Precint: {precinct}</p>
```

Comment:
Also `h2` and `p` tags cannot be inside a button element. Please have a look here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button under "Phrasing Content".